From 3736e5234e04cea4dc722247a148b37c6e1884c2 Mon Sep 17 00:00:00 2001
From: Pacho Ramos <pachoramos@gmail.com>
Date: Sat, 27 Oct 2012 10:38:38 +0000
Subject: build: Use gmime-2.6

Force using gmime-2.6 usage to prevent automagic dependencies
now that older versions are deprecated.

https://bugzilla.gnome.org/show_bug.cgi?id=686975
---
diff --git a/configure.in b/configure.in
index ba40b2e..ea539d2 100644
--- a/configure.in
+++ b/configure.in
@@ -75,15 +75,9 @@ AC_ARG_ENABLE(gmime-i-know-what-im-doing,
 
 if test "x$enable_gmime" = "xyes" ; then
    PKG_CHECK_MODULES(GMIME,  
-		     gmime-2.4,
-                     [have_gmime=yes GMIME=gmime-2.4], 
-                     [have_gmime=no])
-   if test "x$have_gmime" != "xyes" ; then
-     PKG_CHECK_MODULES(GMIME,  
 		       gmime-2.6,
 		       [have_gmime=yes GMIME=gmime-2.6], 
 		       [have_gmime=no])
-   fi
 
    if test "x$have_gmime" = "xyes" ; then
       AC_SUBST(GMIME, $GMIME)
--
cgit v0.9.0.2
